app/javascript/entrypoints/pdc/pdc_ui_loader.es6

```diff
@@ -8,5 +9,6 @@ export default class PdcUiLoader {
   setup_fileupload_validation() {
     new MaximumFileUpload("patch_pre_curation_uploads","file-upload");
     new MaximumFileUpload("pre_curation_uploads","btn-submit");
+    new CopytoClipboard();
```

Comment:
There will be an eslint warning about using a constructor just for side effects.